### ISSUE TITLE
Group gripper cannot work in Gazebo + Moveit

### ISSUE THREAD

**Issue by HappySamuel** [Kinovarobotics/kinova-ros#318]
## Description
KDL kinematics plugin complains that the gripper is not a chain, but i am using the default setting from the master source.

![Screenshot from 2020-10-05 18-09-11](https://user-images.githubusercontent.com/18137963/95067160-f3cea780-0735-11eb-9e56-e8bb16ecb699.png)

![Screenshot from 2020-10-05 18-12-34](https://user-images.githubusercontent.com/18137963/95067395-5e7fe300-0736-11eb-97fd-6d68d9ecf8d4.png)

Error 1: Group 'gripper' is not a chain

Error 2: Kinematics solver of type 'kdl_kinematics_plugin/KDLKinematicsPlugin' could not be initialized for group 'gripper'

Error 3: Kinematics solver could not be instantiated for joint group gripper.

## Version

ROS distribution : ROS Kinetic + Gazebo 9.0

Branch and commit you are using : master

## Steps to reproduce

1. roslaunch kinova_gazebo robot_launch.launch kinova_robotType:=j2s6s300
2. roslaunch j2s6s300_moveit_config j2s6s300_gazebo_demo.launch

## Expected behavior
Gripper control available with moveit.

Need help/guidance on this, how to get the gripper controlled by moveit as well. I am following the tutorial, but found this error.

Best,
Samuel

**Comment by soongjian** [comment]
I have the same problem. Have you solved it?

**Comment by HappySamuel** [comment]
Hi @soongjian 

I yet to get it work. Hope @KinovaCustomerSupport can guide us on how to get it work.

Best,
Samuel

**Comment by alexvannobel** [comment]
Hi @soongjian and @HappySamuel ,
Sorry for the long delay.
I'm looking into this right now.
I can reproduce the error and it also seems the j2s**7**s300 works fine on my end, so I'm looking into package-specific errors by comparing the two.
I'll keep you posted.
Alex

**Comment by alexvannobel** [comment]
I've created a PR, it seems like the implementation differed because a kinematics solver was defined for the gripper, which does not make any sense and createds the error.
I was able to launch the simulation, send the gripper to the "Open" position and then to the "Close" position with the change.
The merge will close the issue but don't hesitate to reopen if you still experience problems.

Best,
Alex
